src/_help/specification-support/extensions.md

```diff
@@ -24,6 +24,46 @@ This custom property allows you to identify some components of your
 documentation as beta. Find out more in our [dedicated
 section](/help/specification-support/doc-beta).
 
+## Give precision to the state of your features (`x-state`) 
+
+This custom property can be added inside an operation or a property to identify it with a custom tag.
+The `x-state` property is a string.
+
+### Example usage
+
+Here under is an example of an operation and a property with a `x-state`.
+
+```yaml
+paths:
+  /diffs:
+    post:
+      tags: ["x-state"]
```

Comment:
I think this is confusing (to have a tag named `x-state` in the definition). I'd recommend to remove the line altogether.